clients/src/test/java/org/apache/kafka/clients/consumer/internals/MockGroupRebalanceConfig.java

```diff
@@ -0,0 +1,6 @@
+package org.apache.kafka.clients.consumer.internals;
+
+import org.apache.kafka.clients.GroupRebalanceConfig;
+
+public class MockGroupRebalanceConfig extends GroupRebalanceConfig {
```

Comment:
This mock seems not necessary as it was not used anywhere.

And even in some unit test cases, I think we can just use a dummy config?